.github/workflows/publish.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish
+
+on:
+  push:
+    tags: '*'
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
+    - name: Install dependencies and build
+      run: |
+        python -m pip install --upgrade pip wheel
+        pip install twine
+        python setup.py sdist bdist_wheel
+    - name: Publish to PyPI
+      uses: pypa/gh-action-pypi-publish@master
+      with:
+        user: __token__
+        password: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
added PyPI API token on repository secret

Reply:
and publish is tested in the PR below

https://github.com/hirosassa/gokart/pull/5
https://test.pypi.org/project/gokart/0.0.0.dev0/